core/common/src/main/scala/org/typelevel/otel4s/meta/InstrumentMeta.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2022 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.otel4s.meta
```

Comment:
Should the module name be `meta` maybe? Typically the module name and package name are somehow matching.

Reply:
Also, is this file misplaced? Or maybe I'm very confused by the source reorganizaiton 🤔

Reply:
Ah, sorry, ignore me 😂